source/algorithms.tex

```diff
@@ -11156,6 +11156,16 @@
 are destroyed in an unspecified order
 before allowing the exception to propagate.
 
+\pnum
+\begin{note}
+When new objects are created by
+the algorithms specified in \ref{specialized.algorithms},
+the lifetime ends for any existing objects
+(including potentially-overlapping subobjects \ref{intro.object})
+in storage that is reused \ref{basic.life}.
+Subsequent operations on these objects can result in undefined behavior.
```

Comment:
It's ambiguous whether "these objects" refers to the new ones created by the algorithms, or the existing ones. I don't think we need to say that using objects outside their lifetime is undefined.

Reply:
Agreed. Reverted to your suggested wording. I tried to seek how to describe "existing objects whose storage were reused" in short but failed.